RFCs/0035-partiql-window-function.md

```diff
@@ -0,0 +1,1224 @@
+
+Similarly, the result in nested data format will be:
+```
+<<
+ -- Row 1
+  {
+    'trade_date' : 2022-09-30,
+    'ticker' : 'AMZN', 
+    'price', 113.00
+    -- partition for row 1
+    'partition' : 
+      [
+          <<
+            'trade_date' : 2022-09-30,
+            'ticker' : 'AMZN', 
+            'price' : 113.00
+          >>,
+          <<
+            'trade_date' : 2022-10-03,
+            'ticker' : 'AMZN', 
+            'price' : 115.88
+          >>,
+          <<
+            'trade_date' : 2022-10-04,
+            'ticker' : 'AMZN', 
+            'price' : 121.09
+          >>
+      ]
+  }    
+  -- rest of the data skiped
+  ...  
+>>
+```
+Example 1.1.4:
+
+Window specification has ORDER BY sub-clause
+
+`<func> OVER (ORDER BY price) -- has ORDER BY sub-clause`
+
+![Example 1.1.4 Graph](./0035-partiql-window-function/Example_1_3.png)
+
+```
+<<
+ -- Row 1
+  {
+    'trade_date' : 2022-09-30,
+    'ticker' : 'AMZN', 
+    'price', 113.00
+    -- partition for row 1
+    'partition' : 
+      [
+          <<
+            'trade_date' : 2022-09-30,
+            'ticker' : 'GOOG', 
+            'price' : 96.15
+          >>,
+          <<
+            'trade_date' : 2022-10-03,
+            'ticker' : 'GOOG', 
+            'price' : 99.30
+          >>,
+          <<
+            'trade_date' : 2022-10-04,
+            'ticker' : 'GOOG', 
+            'price' : 101.04
+          >>,
+          <<
+            'trade_date' : 2022-09-30,
+            'ticker' : 'AMZN', 
+            'price' : 113.00
+          >>,
+          <<
+            'trade_date' : 2022-10-03,
+            'ticker' : 'AMZN', 
+            'price' : 115.88
+          >>,
+          <<
+            'trade_date' : 2022-10-04,
+            'ticker' : 'AMZN', 
+            'price' : 121.09
+          >>
+      ]
+  }    
+  -- rest of the data skiped
+  ...  
+>>
+```
+
+Example 1.1.5:
+
+Window specification has both PARTITION BY AND ORDER BY sub-clause
+
+Notice that to demonstrate the difference visually, the result will be sorted in descending order.
+
+`<func> OVER (PARTITION BY ticker ORDER BY price DESC)  -- has PARTITION BY and ORDER BY sub-clause`
+
+![Example 1.1.5 Graph](./0035-partiql-window-function/Example_1_4.png)
+
+```
+<<
+ -- Row 1
+  {
+    'trade_date' : 2022-09-30,
+    'ticker' : 'AMZN', 
+    'price', 113.00
+    -- partition for row 1
+    'partition' : 
+      [
+          <<
+            'trade_date' : 2022-10-04,
+            'ticker' : 'AMZN', 
+            'price' : 121.09
+          >>,
+          <<
+            'trade_date' : 2022-10-03,
+            'ticker' : 'AMZN', 
+            'price' : 115.88
+          >>,
+          <<
+            'trade_date' : 2022-09-30,
+            'ticker' : 'AMZN', 
+            'price' : 113.00
+          >>
+      ]
+  }
+  -- rest of the data skiped
+  ...  
+>>
+```
+
+#### WINDOWED Clause 
```

Comment:
Going through this section — with reading the previous section — I still see a gap. Maybe if we show the usage of the nested data in one of the examples we can fill this gap, but as of now, the previous section seems disconnected from the whole RFC.

Reply:
Reorganized the work a bit based on our discussion. 

Move the section to the Appendix.